docs/pages/database-access/auto-user-provisioning/database-object-permissions.mdx

```diff
@@ -0,0 +1,205 @@
+    scope:
+      database_names:
+      - 'horizon'
+      - 'horizon_v2'
+      schema_names:
+      - 'application'
+      - 'data_import'
+  # Add `dept: hr` label for respective tables.
+  - add_labels:
+      dept: hr
+    match:
+      table_names:
+      - '*'
+    scope:
+      schema_names:
+      - 'recruitment'
+      - 'salaries'
+      - 'pto'
+      - 'hr_scratchpad'
+version: v1
+```
+
+Save the rule to a file and execute `tctl create -f ownership_nonprod.yaml` to create it in Teleport.
+
+## Database permissions in roles
+
+To grant user permissions during a database connection, the user must be associated with a role that meets specific criteria:
+- `spec.allow.db_labels` must match the database labels of particular database.
+- Database user auto-provisioning should be enabled (`spec.options.create_db_user_mode` not set to `off` or `spec.options.create_db_user: true`).
+- The label key/value pairs in `spec.allow.db_permissions.match` should correspond to the labels on the specific database object.
+
+The labels on the table must be matched with an appropriate role. Here's an example of a role that utilizes the `dept` label, applied by the `ownership_nonprod` rule, granting read-only access to HR records in the database. The `hr_scratchpad` table is further made editable. On the other hand, any objects labeled `dept: sales` are made unavailable by removing all permissions a user may have received for them. The wildcard permissions are only allowed in the `deny` part of the spec (`spec.deny.db_permissions`).
+
+```yaml
+kind: role
+metadata:
+  name: dept_hr_permissions
+spec:
+  allow:
+    db_labels:
+      '*': '*'
+    db_names:
+    - '*'
+    db_permissions:
+    # defult permission: read-only
+    - match:
+        object_kind: table
+        dept: hr
+      permissions:
+      - SELECT
+    # extra permissions for select tables
+    - match:
+        object_kind: table
+        dept: hr
+        name: hr_scratchpad
+      permissions:
+      - SELECT
+      - UPDATE
+      - DELETE
+      - INSERT
+  deny:
+    db_permissions:
+    # explicitly disallow any interaction with `dept: sales` tables.
+    - match:
+        dept: sales
+      permissions:
+      - '*'
+  options:
+    create_db_user_mode: keep
+version: v7
+```
+
+## Permissions lifecycle and consistency
+
+A user can maintain multiple simultaneous connections to the same database. All connections must possess identical permissions; otherwise, a new connection will be rejected. Upon the termination of the last active connection, all user permissions are automatically revoked.
+
+## Troubleshooting
+
+
+### Checking the logs
+
+To diagnose the import process, refer to the database service logs to find details such as the number of objects fetched from the database, the number of imported objects (the difference comprising objects not matched by any import rule), and finally, the number of objects for which the user has been granted permissions.
```

Comment:
Does "database service" refer to the Teleport Database Service? If so, we should say "Teleport Database Service". If not, we should be more specific that these are the logs of the proxied database.